.github/workflows/build-container-and-test-aarch64.yml

```diff
@@ -0,0 +1,83 @@
+name: Lint, build gProfiler container and run tests on aarch64
+
+on: pull_request
+
+jobs:
+  build-container-aarch64:
+    runs-on:
+      - self-hosted
+      - public
+      - ARM64
+
+    steps:
+    - name: change pycache permissions
```

Comment:
Why is this step required?

Reply:
I'm not sure whether its aarch64 or newest ubuntu, but packages installed by python3 -m pip are visible only for the root user. Easiest workaround I found (after failing with PYTHONPATH) is adding sudo to the test script. Then, after installation of pip packages there are some pycache dirs with incorrect permissions. This step changes it back.

Reply:
I experienced identical problems on our aarch64 ubuntu vm, so I assumed it always occurs with such setup combination.